src/material/snack-bar/snack-bar-container.ts

```diff
@@ -152,6 +152,12 @@ export class MatSnackBarContainer extends BasePortalOutlet implements OnDestroy
     // where multiple snack bars are opened in quick succession (e.g. two consecutive calls to
     // `MatSnackBar.open`).
     this._animationState = 'hidden';
+
+    // Mark this element with a 'dismissing' attribute to indicate that the snackbar has
+    // been dismissed and will soon be removed from the DOM. This is used by the snackbar
+    // test harness.
+    this._elementRef.nativeElement.setAttribute('exit', '');
```

Comment:
should we call it `mat-exit`? also the comment says `dismissing`

Reply:
Yeah, I debated this with myself- I figured the prefix wouldn't be strictly necessary since we control the snackbar container and only add it once it's dismissed, but it's fair to say that people might _still_ see it somehow and wonder where it comes from. Added (and fixed the comment).